src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,88 @@
+package wolox.training.models;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Collection;
+import java.util.List;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false, unique = true)
+    private String username;
+
+    @Column(nullable = false, unique = true)
+    private String name;
+
+    @Column(nullable = false, unique = true)
+    private LocalDate birthdate;
+
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
This is wrong. With this you're saying that any book that get's put inside this collection, should be persisted. And if it's removed, it should be deleted from the database. Given a book can exist without being assigned to any user, the only thing you need here is Cascade.MERGE and Cascade.REFRESH, which maintain the join table updated accordingly, and that's it :)